hyundai_canfd.dbc

```diff
@@ -470,7 +470,10 @@ BO_ 676 CAM_0x2a4: 24 XXX
  SG_ BYTE4 : 32|8@1+ (1,0) [0|255] "" XXX
  SG_ BYTE5 : 40|8@1+ (1,0) [0|255] "" XXX
  SG_ BYTE6 : 48|8@1+ (1,0) [0|255] "" XXX
- SG_ BYTE7 : 56|8@1+ (1,0) [0|255] "" XXX
+ SG_ LEFT_LANELINES : 56|2@1+ (1,0) [0|3] "" XXX
+ SG_ NEW_SIGNAL_1 : 58|2@1+ (1,0) [0|3] "" XXX
+ SG_ RIGHT_LANELINES : 60|2@1+ (1,0) [0|3] "" XXX
+ SG_ NEW_SIGNAL_2 : 62|2@1+ (1,0) [0|3] "" XXX
```

Comment:
Since the Lane Safety setting in vehicle settings disappears, there must be something other than "no lanelines" we're sending, unless no lane lines is not 0, and 0 means something else. Some comments of what you've discovered would go a long way!

Reply:
We noticed that on the Ioniq 6 HDA2, the Lane Safety setting still disappeared even when we didn't block the message to suppress LFA. We confirmed this with and without blocking this message. I suspect the EV6 HDA2 might have the same behavior, so the Lane Safety setting is probably affected by something other than this message.

The signals seem to change on the level of certainty with left/right lanelines. I haven't seen any noticeable pattern with it at the moment. As long as one of the signals is at least `1`, LFA would allow to engage. I will write some comments with my findings. See example below:
![image](https://github.com/commaai/opendbc/assets/47793918/ea68a93b-cba5-41ec-95b8-dfc8afaee1d6)